data/src/test/java/com/microsoft/azure/kusto/data/KustoDateTimeTest.java

```diff
@@ -1,58 +1,70 @@
 package com.microsoft.azure.kusto.data;
 
-import org.json.JSONArray;
-import org.json.JSONObject;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import java.time.Instant;
 import java.time.LocalDateTime;
-import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
 
 public class KustoDateTimeTest {
+
+    private ObjectMapper mapper = null;
+
+    @BeforeEach
+    void setup() {
+    }
+
     @Test
     void KustoResultSet() throws Exception {
-        JSONArray rows = new JSONArray();
-        JSONArray row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.1Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.12Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.123Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.1234Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.12345Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.123456Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.1234567Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.12345678Z"));
-        rows.put(row);
-        row = new JSONArray();
-        row.put(Instant.parse("2022-05-17T00:00:00.123456789Z"));
-        rows.put(row);
+        ObjectMapper mapper = Utils.getObjectMapper();
```

Comment:
The global variable mapper is not defined and a new variable is added that shadows this.  The variable mapper is re-defined  , perhaps have a @ BeforeAll or something ? . There is only test , so you may as well remove the global variable

    void KustoResultSet() throws Exception {
        ObjectMapper mapper = Utils.getObjectMapper();